src/server/utils/processManager.js

```diff
@@ -177,8 +178,10 @@ export default class ProcessManager {
   execute() {
     if (!this.initCommand) return;
 
+    const env = Object.assign({}, process.env);
+    env.Path += (isWin ? ';' : ':') + process.cwd() + '/node_modules/.bin'; // Using process.cwd until a better solution is found for (#5)
```

Comment:
I think this should be `env.PATH`?

```suggestion
    env.PATH += (isWin ? ';' : ':') + process.cwd() + '/node_modules/.bin'; // Using process.cwd until a better solution is found for (#5)
```

Reply:
Oh, sorry I totally forgot, yes that would be for *nix machines but the solution for Windows machines will be more complicated as their environment variables are not case-sensitive and a user might have their `PATH` variable set to `Path` or maybe even `PaTh`. I would need to push a new commit to fix this one

Reply:
Ahh interesting, looks like you'll need to check if `env.path` is set in any case/s and update that specific variable name if it is set. 
Looking forward to seeing your implementation!